dev-requirements.txt

```diff
@@ -1,57 +1,47 @@
 #
-# This file is autogenerated by pip-compile with Python 3.7
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Looks like you have used python 3.11 to create the virtual environment where you ran `nox`. Can you do the same steps using python 3.7?

Reply:
@karthiknadig I am getting an error while installing python 3.7.

Reply:
@karthiknadig Hey I tried installing Python 3.7 again but it's still not working. 

<img width="833" alt="Screenshot 2023-09-23 at 15 32 56" src="https://github.com/microsoft/vscode-pylint/assets/33456779/1e1c3aa8-9fc7-4725-bf44-d1631e66e3eb">

Reply:
you have an activated virtual environment. That environment might be 3.11. You will need to delete the `.venv` directory and then re-create the `.venv` using the `Python: Create Environment` command.